mobify/platform-scaffold
# Fix SFCC Connector

Fixed up a bunch of issues with AMP and the SFCC Connector. The PR will remain configured for SFCC until it is approved, then I will switch it back to the Merlins one. 

 **JIRA**: https://mobify.atlassian.net/browse/AMP-76

## Changes
- Fixed a bunch of browser environment dependencies in the SFCC Connector

## Todos
- [x] Document how to configure for SFCC

## How to test-drive this PR
- Run `npm run dev`
- Check out http://localhost:3000/s/2017refresh/mens for the PLP, and http://localhost:3000/s/2017refresh/11736753.html for the PDP
